cmd/admin/cert.go

```diff
@@ -248,7 +285,9 @@ func (a *admin) serialsFromCertPEM(_ context.Context, filename string) ([]string
 	return []string{core.SerialToString(cert.SerialNumber)}, nil
 }
 
-func cleanSerial(serial string) (string, error) {
+// cleanSerials removes non-alphanumeric characters from the serials and checks
+// that all resulting serials are valid (hex encoded, and the correct length).
+func cleanSerials(serials []string) ([]string, error) {
```

Comment:
Consider moving this function upwards in the file, so that it's not sandwiched between the two functions that use it.

Alternatively if you don't want to move this code, you could move `revokeMalformed` down here.

Reply:
This function is now only used by one function, `Run`, and appears directly after that function.

There was a call in `revokeSerials`, but that got moved up a level so that it applies to both `revokeSerials` and `revokeMalformed`.